nix/nixpkgs/default.src.json

```diff
@@ -1,6 +1,6 @@
 {
   "owner": "NixOS",
   "repo": "nixpkgs",
-  "rev": "3a86856a13c88c8c64ea32082a851fefc79aa700",
-  "sha256": "04dd4g24061p8sw922zjickahqwn3wa9l3j5qllfzp5n6dy9is16"
+  "rev": "9a82a9b5248919805a2400266ebd881d5783df2a",
+  "sha256": "142x1zq3cjadgmvfv0paydlq268pfinllqpq2vl0vxwdiq2nr9iz"
```

Comment:
This is the heart of this change: it looks like Bazel 5.2.0 is simply broken, so I'm bumping to 5.3.2.

Reply:
oh :( https://github.com/bazelbuild/bazel/issues/15682